test/performance/odp_crypto.c

```diff
@@ -80,6 +80,8 @@ static uint8_t test_key64[64] = { 0x01, 0x02, 0x03, 0x04, 0x05,
 typedef struct {
 	const char *name;		      /**< Algorithm name */
 	odp_crypto_session_param_t session;   /**< Prefilled crypto session params */
+	int cipher_in_bit_mode;               /* probed at run time */
+	int auth_in_bit_mode;                 /* probed at run time */
```

Comment:
Use uniform comment style for all members of a struct, if possible.

Reply:
Fixed in v2. I also made the comments hopefully a bit more descriptive.